defaults/main.yml

```diff
@@ -35,10 +38,15 @@ beacon_node_discovery_port: 9000
 beacon_node_listening_port: 9000
 
 # Scraping of metrics done via VPN. Protected by firewall.
+beacon_node_metrics_enabled: true
 beacon_node_metrics_address: '0.0.0.0'
 beacon_node_metrics_port: 9200
+
+beacon_node_rpc_enabled: false
```

Comment:
Maybe a comment explaining why we are disabling it by default?

Reply:
Fixed